src/components/20-molecules/dropdown/ui.test.js

```diff
@@ -11,13 +11,127 @@ test('should render correctly', async t => {
   await t.expect(dropdown.exists).ok();
 });
 
+fixture('Dropdown valid').page(
+  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-valid-checkmark`
+);
+test('should show checkmark', async t => {
+  const dropdownCheckmark = await Selector(() =>
+    document.querySelector(
+      `axa-dropdown[data-test-id="dropdown-valid"] .m-dropdown__valid-icon .m-dropdown__valid-icon-inner-active`
+    )
+  );
+  await t.expect(dropdownCheckmark.exists).ok();
+});
+
+fixture('Dropdown error').page(
+  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-error-message`
+);
+
+test('should show error message and correct colors', async t => {
+  const dropdown = await Selector(() =>
+    document.querySelector(`axa-dropdown[data-test-id="dropdown-error"]`)
+  );
+
+  await t.expect(dropdown.exists).ok();
+
+  const getErrorMessage = ClientFunction(() => {
+    const errorDiv = document.querySelector(
+      'axa-dropdown[data-test-id="dropdown-error"] > .m-dropdown__error'
+    );
+    return errorDiv.innerText;
+  });
+
+  await t
+    .wait(50)
```

Comment:
You use waits almost everywhere. I don't like it. If not explicitly needed, I would propose to not use them, because 20 waits like that will already make the build one second slower, and probably for no reason.

Reply:
Not true. I use them when there is a reason. You could leigitimately ask for that reason, though - that would be a fair point.

Reply:
OK, will document reasons for wait as comments everywhere - or remove if no such reason can be found.

Reply:
done